### ISSUE TITLE
Your equilibrium plot

### ISSUE THREAD

**Issue by JustinJaniszewski** [ubsuny/23-Homework7G1#80]
Does your equilibrium plot look like the original shape or sort of random.

**Comment by tirthbha** [comment]
@JustinJaniszewski You mean coordinate positions?

**Comment by AhmedCode99** [comment]
This is interesting and somewhat useful

**Comment by tirthbha** [comment]
> I suppose. I mean your plot. You dont really need to analyze the points. Looking at your plot, does it look random or the expected shape **after** you optimize

you are right the positions are just for visualization. for the minimized potential  there is an another graph
